### PR TITLE
Fix: load component cue action  can set patchedApp override default app

### PR DESCRIPTION
**Pull request by wangyikewxgm**
### Description of your changes

1.  load component cue action  can set patchedApp override default app
2. export some multiCluster cue action, let user can specify user defined workflow.

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Comment by wangyikewxgm** [comment]
@leejanee @Somefive /cc

File: pkg/stdlib/op.cue [hunk 1]
```diff
@@ -107,6 +107,12 @@ import (
 
 #ApplyEnvBindApp: multicluster.#ApplyEnvBindApp
 
+#LoadPolicies: oam.#LoadPolicies
+
+#MakePlacementDecisions: multicluster.#MakePlacementDecisions
+
+#PatchApplication: multicluster.#PatchApplication
+
```
Comment (leejanee) [inline pkg/stdlib/op.cue:115]: These codes are not related to the current pr, right?

File: pkg/workflow/providers/oam/apply.go [hunk 2]
```diff
@@ -139,7 +141,22 @@ func lookUpValues(v *value.Value) (*common.ApplicationComponent, *value.Value, s
...
+	// if specify `app`, use specified application otherwise use default application fron provider
+	appSettings, err := v.LookupValue("app")
+	if err != nil {
+		if strings.Contains(err.Error(), "not exist") {
+			app = p.app
+		} else {
+			return err
+		}
+	} else {
+		fmt.Println(appSettings.String())
```
Comment (leejanee) [inline pkg/workflow/providers/oam/apply.go:154]: fmt

Comment (wangyikewxgm) [reply pkg/workflow/providers/oam/apply.go]: fixed

**Review by leejanee — APPROVED** [review]
